apps/website/src/pages/join-us/[ashbyIdOrCmsSlug].tsx

```
@@ -0,0 +1,208 @@
+          </Head>
+          <HeroSection>
+            <HeroH1>{job.title}</HeroH1>
+            {job.location && <HeroH2>{job.location}</HeroH2>}
+            <HeroCTAContainer>
+              <CTALinkOrButton url="#application-form-anchor">Apply Now</CTALinkOrButton>
+            </HeroCTAContainer>
+          </HeroSection>
+          <Breadcrumbs route={currentRoute} />
+          <Section className="max-w-3xl">
+            {/* eslint-disable-next-line react/no-danger */}
+            <div className="prose prose-p:text-size-md prose-li:text-size-md prose-p:leading-normal prose-li:leading-normal max-w-none" dangerouslySetInnerHTML={{ __html: job.descriptionHtml }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)